.circleci/workflows.yml

```diff
@@ -297,9 +301,37 @@ jobs:
           git commit -m "Auto-generated docs based on beamer ${CIRCLE_SHA1}"
           git push origin gh-pages
 
+  publish-frontend:
+    executor: node/default
+    working_directory: ~/beamer/frontend
+    environment:
+      REPO_URL: git@github.com:beamer-bridge/frontend-deployment.git
+      TMP_DIR: /tmp/frontend-deployment
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - fa:3a:6e:26:54:04:d8:58:3f:63:7c:3d:00:a1:d7:c2:be
+      - run: |
+          mkdir -p ~/.ssh
```

Comment:
I think `-p` does nothing here and can be removed. Would make sense if there were any subdirectories in `~/.ssh`

Reply:
Had a failing run without the option